draft-ietf-tls-tls13.md

```diff
@@ -951,14 +962,29 @@ message MAY be fragmented across several records).
 
        enum {
            reserved(20), alert(21), handshake(22),
-           application_data(23), (255)
+           application_data(23), handshake_padded(25),
+           application_data_padded(26), (255)
        } ContentType;
 
        struct {
+           uint16 padding_length;
+           opaque fragment[TLSPlaintext.length - padding_length];
+           opaque padding[padding_length];
+       } PaddedFragment;
```

Comment:
This will still add needless complexity if the packet fragments don't align with handshake message borders (ie the padding is in the middle of a handshake message), and the handshake_hash needs to be calculated (as the handshake layer may not know anything about the packet layer).